back/api/serializers.py

```diff
@@ -73,6 +73,14 @@ class Meta:
 
 
 class IdentitySerializer(serializers.ModelSerializer):
+
+    def validate(self, data):
+        if data.get('company_name', '').strip() != '':
```

Comment:
Hmmmm, currently, sometimes lambda users also have something inside the `company_name` attribute. I am not sure about the new version, but it might be necessary to specify if the user is part of a company or not in an additional attribute (up to you to choose if necessary or not...)

Reply:
Okay I reverted that... I don't want to add a field for company or not, we will just add the IDE field on front saying it's mandatory for companies. We will see then...